### DISCUSSION TITLE
2023/04/25/%E7%A7%BB%E5%8A%A8%E6%8D%AE%E7%82%B9%E8%AE%BE%E7%BD%AE/

### DISCUSSION THREAD

**Discussion by giscus** [FloranceYeh/FloranceYeh.github.io#6] [Announcements]
# 2023/04/25/%E7%A7%BB%E5%8A%A8%E6%8D%AE%E7%82%B9%E8%AE%BE%E7%BD%AE/

It's Florance!

https://floranceyeh.netlify.app/2023/04/25/%E7%A7%BB%E5%8A%A8%E6%8D%AE%E7%82%B9%E8%AE%BE%E7%BD%AE/

**Comment by ERA-YES** [comment]
留个脚印👣
